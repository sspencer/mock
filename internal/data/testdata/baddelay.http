### Bad status
# @status=204
# @delay=zero
DELETE /bread/1
