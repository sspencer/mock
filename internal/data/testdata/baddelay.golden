invalid duration, line 3: # @delay zero